Tooling.md

```diff
@@ -44,6 +44,12 @@ The tooling we expect to support includes:
   - Instrumentation-based.
 * Process dump: local variables, call stack, heap, global variables, list of
   threads.
+* JavaScript+WebAssembly size optimization tool. Some developers have huge
```

Comment:
lgtm, but all the other bullet points are a name + `:`. Perhaps change the first `.` to `:`? In which case, maybe also edit to remove the later `:` to avoid repetition.